platform-common/src/main/java/com/softicar/platform/common/testing/Asserts.java

```diff
@@ -152,6 +152,11 @@ public static <T> T assertOne(Iterable<T> objects) {
 		return assertCount(1, objects).get(0);
 	}
 
+	public static <T> T assertOne(Optional<T> optional) {
+
+		return optional.orElseThrow(() -> new AssertionError("Expected optional to not be empty."));
```

Comment:
I think that custom String could be omitted as well.

Reply:
I don't understand. Please explain.

Reply:
I mean to simply use `new AssertionError()` instead of `new AssertionError("Expected optional to not be empty.")`